metaci/release/utils.py

```diff
@@ -96,8 +109,10 @@ def send_release_webhook(project_config, release, config_item=None):
 
     payload = {
         "case_template_id": release.change_case_template.case_template_id,
-        "package_name": project_config.project__package__name,
-        "version": project_config.get_version_for_tag(tag),
+        "package_name": release.repo.name,  # Need to figure out.
+        "version": release.repo.latest_release.git_tag.strip(
+            f"{release.repo.release_tag_regex}"
+        ),  # Need to see if this is valid.
```

Comment:
Just use release.git_tag, not release.repo.latest_release.git_tag. No one should be running the plan for an old release, but if they do we don't want to tell GUS that it's the latest release number.

And also, as written this will break if release.git_tag is null (release engineer forgot to fill in the field when creating the release). We need to either make the field required or make this code more robust so that it can raise an error that says what happened instead of just `AttributeError: NoneType object has no attribute strip`.